demosplan/DemosPlanCoreBundle/Repository/EmailAddressRepository.php

```diff
@@ -64,10 +64,12 @@ public function deleteOrphanEmailAddresses(): int
         return $connection->exec(
             'DELETE e'
             .' FROM email_address AS e'
-            .' LEFT JOIN procedure_agency_extra_email_address      AS p  ON p.email_address_id = e.id'
-            .' LEFT JOIN maillane_allowed_sender_email_address    AS p2 ON p.email_address_id = e.id'
-            .' WHERE p.procedure_id  IS NULL'
-            .' AND   p2.procedure_id IS NULL'
+            .' LEFT JOIN procedure_agency_extra_email_address  AS p  ON p.email_address_id = e.id'
+            .' LEFT JOIN maillane_allowed_sender_email_address AS m  ON m.email_address_id = e.id'
+            .' LEFT JOIN support_contact                      AS sc ON sc.email_address = e.id'
+            .' WHERE p.procedure_id   IS NULL'
+            .' AND   m.procedure_id   IS NULL'
+            .' AND   sc.email_address IS NULL'
         );
     }
```

Comment:
If not already existing, this seems like something that definitely  should be covered by some tests.